src/routes/api/trips.routes.js

```diff
@@ -469,4 +470,41 @@ router.patch('/:tripId', [checkUser, isProfileUpdated,
   isUuidParamValid, isTripExist, areYouTripOwner, isTripRejected,
   isTripApproved, isEditTripRequestValid],
 tripsController.editTrip);
+
+/**
+ * @swagger
+ *
+ * /api/v1/trips/stats:
+ *   post:
+ *     security: []
+ *     summary: Travel requests statistics
+ *     description: Show travel requests statistics of a user in a given timeframe
+ *     tags:
+ *       - Trips
+ *     requestBody:
+ *       content:
+ *         application/json:
+ *           schema:
+ *             type: object
+ *             properties:
+ *               from:
+ *                 type: string
+ *               until:
+ *                 type: string
+ *     produces:
+ *         application/json:
+ *           schema:
+ *             type: object
+ *             properties:
+ *               status:
+ *                 type: integer
+ *               message:
+ *                 type: string
+ *               data:
+ *                 type: object
+ *     responses:
+ *       200:
+ *         description: Your travel requests statistics between the given dates
+ */
+router.post('/stats', [checkUser, validateStats], tripsController.tripStatistics);
```

Comment:
@AbdoulNuru Bravo for making this functionality happens;
I just have a quick few suggestions for best practice as follows:

``POST /api/v1/trips/stats`` should be`` GET`` request plus query strings or params for ``from`` and ``until ``dates because; GET request is used to retrieve data from server and has no other effect whereas POST is used to create a new resource of posted data in the request body. Read this resource for [GET request](https://www.w3.org/Protocols/HTTP/Methods.html#z47)  and this one for [POST request](https://reqbin.com/Article/HttpPost) as well.
If you wanna implenet it in that way conside my below suggestions to have a clue.
```suggestion
router.get('/stats/:from/:until', [checkUser, validateStats], tripsController.tripStatistics);
```
Don't hestate to hit me up for any concern like `` a PRO``

Reply:
@hakoemmy Thank you, I will look into your suggestions.

Reply:
instead of doing this `/stats/:from/:until` i think it would be better if u used query strings which doesn't require you to specify them on the route 
so that they can be passed on as follows `/stats?from=date&unti=date`

Reply:
@blaise82 I don't think there is a downside in using either query string or params, so I think that's just personal preference.

Reply:
@AbdoulNuru Thank you for implementing the feedback in a good way